src/test/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileWriterImplTest {
+    private static FileWriter fileWriter;
+    private static String pathToWrite;
+    private static String text;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void initializeFields() {
+        fileWriter = new FileWriterImpl();
+        pathToWrite = "src/test/resources/activities.csv";
+        text = "Test!\\ //" + System.lineSeparator() + "The end";
+    }
+
+    @AfterClass
+    public static void clearFile() {
+        try {
+            Files.deleteIfExists(Paths.get(pathToWrite));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't delete file after test!", e);
+        }
+    }
```

Comment:
@AfterClass should be fater all tests, i guess

Reply:
travis doesn't swear)